react-basic-practice/src/components/CustomerForm/index.tsx

```diff
@@ -22,11 +22,44 @@ const CustomerForm: React.FC<CustomerFormProps> = ({
   onSubmit,
   isLoading,
 }) => {
+  const [errors, setErrors] = useState<{ [key: string]: string }>({});
```

Comment:
create type and update naming. It's not plural

Reply:
i done